CMakeLists.txt

```diff
@@ -114,6 +114,8 @@ find_package(Boost REQUIRED)
 
 add_library(sml INTERFACE)
 
+include_directories(INTERFACE $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>)
```

Comment:
the right way to do it would be to use `target_link_libraries(myBenchmark PUBLIC sml)` for every target.

Reply:
I'd create a function that abstracts the following:
```
    add_executable(test_actions_defer actions_defer.cpp)
    target_link_libraries(test_actions_defer PUBLIC sml) 
    add_test(test_actions_defer test_actions_defer)
```